<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoForm</class>
 <widget class="QWidget" name="InfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="ui_twBaseInfo">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
         <row>
          <property name="text">
           <string>Name</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Title</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Content Type</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Hide</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Private</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Own</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Date</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Modified Date</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Seq</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage2">
      <attribute name="title">
       <string>Summary</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="ui_pbSummaryReset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ui_pbSummaryUpdate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="ui_pteSummary">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage3">
      <attribute name="title">
       <string>Extended Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="ui_pbExtDataReset">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ui_pbExtDataUpdate">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="ui_pteExtData">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_pteExtData</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>ui_pbExtDataReset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pteExtData</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>ui_pbExtDataUpdate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pteSummary</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>ui_pbSummaryReset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pteSummary</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>ui_pbSummaryUpdate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>updateSummary()</signal>
 </slots>
</ui>
